script/import_forecasts.rb

```diff
@@ -0,0 +1,62 @@
+require "optparse"
```

Comment:
Big fan of using scripts, rather than Rake BTW - may nick this approach for my import work 👍

Reply:
Can we get some tests for this script though? The lack of tests makes me a bit nervous

Reply:
Again this was a trade-off of urgency vs return on investment. The classes that do the actual work here are tested, and the stuff in this script is mostly argument parsing, printing output, and asking for user input, which are all fairly painful to write tests for.

I'd be interested in knowing what looks risky here -- are there specific things we could write tests for, without exhaustively setting up tests that fake out the whole process of running a shell script? Any of this that could be moved into classes that we write unit tests for, without the complexity of the UI layer?

Reply:
Yeah, fair enough. It's not actually doing too much, so I think it's not really worth the pain of testing